kit/azure/bootstrap/terraform-state/resources.docs-spn.tf

```diff
@@ -0,0 +1,32 @@
+# allow reading terraform state
+
+# important caveat: the current, simplified design of the bootstrap module means that this the docs UAMI can read
```

Comment:
@florianow this is no longer applicable, i.e. there is no more bootstrap SPN credential, right?